index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt A Dog: Home</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400%7CGelasio&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/eb51aaebc6.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+  </head>
+  <body>
+      <header>
+        <div class="header-content-container">
+          <div class="logo-content-container">
+            <img src="images/adoptadog-logo.svg" alt="Adoptadog white logo">
+            <p>Seattle's premiere dog adoption agency</p>
+          </div>
+          <div class="checkout-content-container">
+            <div class="cart-content-container">
+              <img src="images/cart-icon.svg" alt="shopping cart">
+              <p>$1234.56</p>
+            </div>
+          <div class="checkout-button-content-container">
+            <input type="button" name="Checkout" value="Checkout">
+          </div>
+          </div>
+        </div>
+      </header>
+      <nav>
+          <ul>
+            <li><a href="home.html">Home</a></li>
+            <li><a href="dogs.html">Dogs</a></li>
+            <li><a href="blog.html">Blog</a></li>
+          </ul>
+      </nav>
+    <main>
+      <div class="main-content-container">
+        <section class="intro">
+          <img src="images/home-intro.jpg" alt="Person hugging golden retriever">
+          <h1>Adopt A Dog</h1>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+            Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+            Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+            Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+            Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
+          </p>
+        </section>
+        <div class="dog-cards">
+
+
+        <section class="card">
```

Comment:
The only thing I'd do different here is the whole grid of cards is a section, and each card is then an article (or a div) but your idea of having the card class nested i the dog-cards div is a smart method (and the way I did it)